Source/Meadow.Foundation.Core/Sensors/Buttons/PushButton.cs

```diff
@@ -154,11 +154,17 @@ private void DigitalInChanged(object sender, DigitalInputPortEventArgs e)
                 _buttonPressStart = DateTime.MaxValue;
 
                 // if it's a long press, raise our long press event
-                if (pressDuration > LongPressThreshold) this.RaiseLongPress();
+                if (LongPressThreshold > TimeSpan.Zero && pressDuration > LongPressThreshold)
```

Comment:
Should this still happen after RaisePressEnded?

Reply:
Sorry, can you please clarify?